tests/integration/test_migrate.sh

```diff
@@ -25,7 +25,12 @@ if [ "$init" != "systemd" ];then
     exit 0
 fi
 
-dockerPid=$(ps -C docker -o pid=|xargs)
+if ! systemctl is-active docker >/dev/null; then
+     echo "Docker daemon is not running"
+     exit 1
```

Comment:
I think it is better to skip the test instead of failing.  We can skip it with "exit 77"